source/vocab/files-packages-representations.ttl

```diff
@@ -141,3 +142,19 @@
     rdfs:label "height"@en, "höjd"@sv ;
     sdo:domainIncludes :MediaObject ; # TODO: broaden domain later to also apply on Manifest and physical objects
     owl:equivalentProperty sdo:height .
+
+:bitsPerSample a owl:DatatypeProperty ;
+    rdfs:label "bits per sample"@en "bits per sample"@sv ;
+    sdo:domainIncludes :MediaObject ;
+    owl:equivalentProperty exif:bitsPerSample .
+
+:colorSpace a owl:DatatypeProperty ;
+    rdfs:label "color space"@en, "färgrymd"@sv ;
+    sdo:domainIncludes :MediaObject ;
+    owl:equivalentProperty exif:colorSpace .
+
+:dateTimeDigitized a owl:DatatypeProperty ;
```

Comment:
Will this be the same as the value of `:date` on a [`:DigitalReproduction`](https://id.kb.se/vocab/DigitalReproduction) event? I think we need to look at how to harmonize the model around these details. (We _may_ define an `owl:propertyChainAxiom` for this shorthand. The important thing is to ensure that we don't have different ways of expressing the same information; to avoid making it hard to know where to put and look for the details.)

Reply:
I think it might be expressing the same value. Meaning, the datetime is the moment in which the digitized image was created and is unchanged by modifications to the image. If it is the the moment in which the camera shot the picture or if it the moment in which the digital time was created, I cannot say for sure. Seems to be hard to differentiate the two in the modern age. But if a DigitalReproduction event *is* the moment a digital file is made, then we are talking about the same thing. 

Is the term 'dateTimeDigitized' superflouous in that case?

Reply:
Thanks for the input, it took a bit of time to come back to this but now we have checked with our consumers and seeing they are specifically interested in date of digitisation per image file, I think it is safer to introduce the new term than to introduce possible confusion in the existing one that, as you say, is not as precise in its name.
Would you agree with that?

Reply:
I have found ImageBitDepth in vocab/enums.ttl

"""
:ImageBitDepth a owl:Class;
    rdfs:label "Image bit depth"@en, "Bildens bitdjup"@sv;
    rdfs:subClassOf :DigitalCharacteristic;
    owl:equivalentClass bflc:ImageBitDepth .
"""

I am unsure about the domain of the term and whether I need to add something in order for it to also describe MediaObjects now. Help is appreciated @niklasl